pandas/core/arrays/datetimes.py

```diff
@@ -1036,14 +1035,7 @@ def normalize(self):
                        '2014-08-01 00:00:00+05:30'],
                        dtype='datetime64[ns, Asia/Calcutta]', freq=None)
         """
-        if self.tz is None or timezones.is_utc(self.tz):
```

Comment:
this fastpath looks like it was removed in favor of the loop in normalize_i8_timestamps. cc @jbrockmendel